src/lib/plugin.js

```diff
@@ -410,8 +413,7 @@ class FiveBellsLedger extends EventEmitter2 {
       throw new ExternalError('Unable to determine current balance')
     }
     const ledgerBalance = new BigNumber(res.body.balance)
-    const integerBalance = ledgerBalance.shift(this.ledgerContext.getInfo().scale)
-    return integerBalance.toString()
+    return ledgerBalance.toString()
```

Comment:
good catch, fixed. I also removed some console.log statements that were still in there.